apps/frontend/src/app/dashboard/dashboard.component.html

```diff
@@ -12,11 +12,20 @@ <h1>{{ 'DASHBOARD' | translate }}</h1>
   </mat-grid-tile>
 </mat-grid-list>
 
+<h2>{{ 'DASHBOARDS.LAST_SESSIONS' | translate }}</h2>
 <ccn-session-table
   [sessionList]="sessionList"
   style="margin-top: 1em; width: 100%"
 ></ccn-session-table>
 
+<h2 style="margin-top: 2rem">
+  {{ 'DASHBOARDS.7_DAY_INCIDENCES' | translate }}
+  <mat-icon
+    matTooltip="{{ 'DASHBOARDS.7_DAY_INCIDENCES_TOOLTIP' | translate }}"
+    style="vertical-align: middle; cursor: pointer"
+    >help_outline
+  </mat-icon>
+</h2>
```

Comment:
kannst du bitte alle Stiländerungen in der Scss-Datei machen, die Hauptidee von Angular ist die Trennung von Logik (service), Style (scss) und Template (html)